README.md

```diff
@@ -51,14 +50,11 @@ To setup the project locally read these wiki pages and follow the instructions:
  - [Fork, Clone and Remote](https://github.com/anitab-org/mentorship-backend/wiki/Fork%2C-Clone-%26-Remote)
  - [Export Environment Variables](docs/environment-variables.md)
 
-### Run app in Windows
+### Run app locally
 
-The project runs on Python 3.
+The project runs on Python 3.9. We use [Poetry](https://python-poetry.org/) to manage dependencies.
 
-1. Create a virtual environment:
-```
-virtualenv venv --python=python3
-```
+1. [Install Poetry](https://python-poetry.org/docs/#installation):
 
 2. Activate the virtual environment:
```

Comment:
With step 1 being about poetry, does this step 2 make sense? Since as you said `venv` folder is not a thing?